contracts/scripts/deployParams.json

```diff
@@ -0,0 +1,5 @@
+{
+  "toll": 1,
+  "tollCollector": "0x53C88539C65E0350408a2294C4A85eB3d8ce8789",
+  "tollReceiver": "0x53C88539C65E0350408a2294C4A85eB3d8ce8789"
+}
```

Comment:
A structure like this might be better since it's more flexible and explicit, and allows the deploy script to throw if the supported network doesn't match a key listed here. Then we can also specify the paymaster addresses here instead of in the deploy script

```json
{
  "localhost": {
    "toll": 1,
    "tollCollector": "0x53C88539C65E0350408a2294C4A85eB3d8ce8789",
    "tollReceiver": "0x53C88539C65E0350408a2294C4A85eB3d8ce8789"
  },
  "rinkeby": {
    "toll": 1,
    "tollCollector": "0x53C88539C65E0350408a2294C4A85eB3d8ce8789",
    "tollReceiver": "0x53C88539C65E0350408a2294C4A85eB3d8ce8789"
  }
}
```

Reply:
I like the approach suggested by @mds1.

Reply:
Another good idea.. done